src/python/pants/engine/internals/scheduler.py

```diff
@@ -409,61 +384,74 @@ def execution_request(
             self._scheduler.execution_add_root_select(native_execution_request, subject, product)
         return ExecutionRequest(request_specs, native_execution_request)
 
-    def invalidate_files(self, direct_filenames):
+    def invalidate_files(self, direct_filenames: Iterable[str]) -> int:
         """Invalidates the given filenames in an internal product Graph instance."""
         invalidated = self._scheduler.invalidate_files(direct_filenames)
         self._maybe_visualize()
         return invalidated
 
-    def invalidate_all_files(self):
+    def invalidate_all_files(self) -> int:
         """Invalidates all filenames in an internal product Graph instance."""
         invalidated = self._scheduler.invalidate_all_files()
         self._maybe_visualize()
         return invalidated
 
-    def node_count(self):
-        return self._scheduler.graph_len()
```

Comment:
Unused. The only caller was internal to this class, and we can call `self._scheduler.graph_len()` directly.